pkg/service/connect_service.go

```diff
@@ -181,11 +184,11 @@ func (s *ConnectService) EventStream(
 }
 
 func (s *ConnectService) Notify(n Notification) {
-	s.eventingConfiguration.mu.Lock()
+	s.eventingConfiguration.mu.RLock()
 	for _, send := range s.eventingConfiguration.subs {
 		send <- n
 	}
-	s.eventingConfiguration.mu.Unlock()
+	s.eventingConfiguration.mu.RUnlock()
```

Comment:
Might be better to pull this up and defer it?

Reply:
done